izuma_systest_lib/cloud/websocket_handler.py

```diff
@@ -401,8 +401,7 @@ def open(self):
         _it = threading.Thread(target=self._input_thread, args=(self._api_url, self._api_key),
                                name='websocket_{}'.format(build_random_string(3)))
         _ht = threading.Thread(target=self._handle_thread, name='messages_{}'.format(build_random_string(3)))
-        _it.setDaemon(True)
-        _ht.setDaemon(True)
+        _ht.daemon = True
```

Comment:
Why is this only for the _ht thread and not also the _it thread ?

Reply:
Oh dang. Crossed yes, I looked the lines to be identical. Which they are not. My bad.

Reply:
This has been fixed now.